### ISSUE TITLE
Improvement: iOS Enable bluetooth microphones

### ISSUE THREAD

**Issue by deJong-IT** [csdcorp/speech_to_text#247]
The AudioSession is now only on the defaultSpeaker (Microphone)

```
try self.audioSession.setCategory(AVAudioSession.Category.playAndRecord, options: .defaultToSpeaker)
```

when you change the category options it will also work on bluetooth devices.
```
try self.audioSession.setCategory(AVAudioSession.Category.playAndRecord, options: [.defaultToSpeaker,.allowBluetooth,.allowBluetoothA2DP])
```

**Comment by deJong-IT** [comment]
I created a pullrequest: #248

**Comment by sowens-csd** [comment]
Do you know if this change will require a new privacy description in the plist file?

**Comment by deJong-IT** [comment]
> Do you know if this change will require a new privacy description in the plist file?

I don't think so. As far as I know this is just the same as "Privacy - Microphone Usage Description".

The only Bluetooth related Privacy descriptions I can find are:
`NSBluetoothPeripheralUsageDescription`: "Privacy - Bluetooth Peripheral Usage Description" (Deprecated)
`NSBluetoothAlwaysUsageDescription`: "Privacy - Bluetooth Always Usage Description"
As far as I can see, you only need `NSBluetoothAlwaysUsageDescription` when you access bluetooth directly, like connecting/searching/etc.
But now we just use the play and record session, and we allow bluetooth devices if the users wants it.

**Comment by sowens-csd** [comment]
This is available now in the 5.0.0 release, thanks @deJong-IT.
